### ISSUE TITLE
no field labels on _routes record edit

### ISSUE THREAD

**Issue by outflux3** [n00dles/DM_Matrix#41]
There appear to be no labels on the fields for the routes table...

**Comment by n00dles** [comment]
Yes, no easy way to add other options when creating a table using the built in functions. 
I'll ad this later...
